README.md

```
@@ -17,15 +16,15 @@ device memory are allocated. For example, using "pinned" host memory for asynchr
 host <-> device memory transfers, or using a device memory pool sub-allocator to reduce the cost of
 dynamic device memory allocation.
 
-The goal of the RAPIDS Memory Manager (RMM) is to provide:
+The goal of RMM is to provide:
 - A common interface that allows customizing memory allocation on device and host
 - A collection of [implementations](#available-resources) of the interface
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)